src/components/movie-preview-modal.jsx

```diff
@@ -115,6 +141,29 @@ export const MoviePreviewModal = ({ whiteButton, buttons, howToWatch }) => {
             </Button>
           </div>
         )}
+
+        {/* buttons for publishing drafts in submit route */}
+        {publishButtons && (
```

Comment:
Great job with the difference in if a user is an admin or if they are just the creator.